azure-pipelines/release.yml

```diff
@@ -20,23 +9,24 @@ steps:
     downloadPath: '$(Agent.TempDirectory)'
 
 - task: CopyFiles@2
-  displayName: 'Copy build drop artifacts to: $(Build.SourcesDirectory)/artifacts/package/artifacts/package'
+  displayName: 'Copy build drop artifacts to: $(Build.ArtifactStagingDirectory)'
   inputs:
     SourceFolder: '$(Agent.TempDirectory)/drop'
-    TargetFolder: '$(Build.SourcesDirectory)/artifacts/package'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
 - script: |
-    cd $(Build.SourcesDirectory)/artifacts/package
+    cd $(Build.ArtifactStagingDirectory)
```

Comment:
Same as above. You need the complete path- staging directory doesn't redirect you to the artifacts/package folder.